lib/util.ml

```diff
@@ -101,6 +101,12 @@ let bigint_to_yojson (b:bigint) : Yojson.Safe.t =
 let bigint_of_yojson (json:Yojson.Safe.t) =
   Ok (Bigint.of_string (Yojson.Safe.to_string json))
 
+let show_bigint (b:bigint) : string =
+  Bigint.to_string b
+
+let pp_bigint fmt b = 
+  Format.pp_print_string fmt (Bigint.to_string b)
+
```

Comment:
Does this belong in bitstring.ml instead?

Reply:
It's needed here because of the way the ppx_deriving works.